src/components/Banner.js

```diff
@@ -0,0 +1,56 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import Hoverable from './Hoverable';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import RenderHTML from './RenderHTML';
+import Text from './Text';
+import styles from '../styles/styles';
+import * as StyleUtils from '../styles/StyleUtils';
+import getButtonState from '../libs/getButtonState';
+
+const propTypes = {
+    /** Text to display in the banner. */
+    text: PropTypes.string.isRequired,
+
+    /** Should this component render the text as HTML? */
+    shouldRenderHTML: PropTypes.bool,
+};
+
+const defaultProps = {
+    shouldRenderHTML: false,
+};
+
+const Banner = props => (
+    <Hoverable>
+        {isHovered => (
+            <View style={[
+                styles.flexRow,
+                styles.alignItemsCenter,
+                styles.p5,
+                styles.borderRadiusNormal,
+                isHovered ? styles.activeComponentBG : styles.hoveredComponentBG,
+            ]}
+            >
+                <View style={[styles.mr3]}>
+                    <Icon
+                        src={Expensicons.Exclamation}
+                        fill={StyleUtils.getIconFillColor(getButtonState(isHovered))}
+                    />
+                </View>
+                {
+                    props.shouldRenderHTML
```

Comment:
NAB, alternatively just always render HTML because plain text run through an HTML renderer is just text anyway